keras_nlp/tokenizers/unicode_character_tokenizer.py

```diff
@@ -45,6 +45,10 @@ class UnicodeCharacterTokenizer(tokenizer.Tokenizer):
             One of The encoding of the input text. Defaults to "UTF-8".
         output_encoding: One of ("UTF-8", "UTF-16-BE", or "UTF-32-BE").
             The encoding of the output text. Defaults to "UTF-8".
+        vocabulary_size: Set the vocabulary `vocabulary_size`,
+            by clamping all codepoints to the range [0, vocabulary_size).
+            Effectively this will make the `vocabulary_size - 1` id the
+            the OOV token.
 
     Examples:
```

Comment:
It might be nice to add an example for this? As a lot of real work use case will probably want to add a cap.

I'm not sure where to draw the cutoff for an example but here's a browser. https://codepoints.net/basic_multilingual_plane

Maybe we just show passing `vocaublary_size=592` for support latin + latin extension characters.

Reply:
Yeah this seems like a good addition!